include/bout/mesh.hxx

```diff
@@ -609,12 +609,12 @@ class Mesh {
   typedef BoutReal (*flux_func)(stencil&, stencil &);
 
   /// Transform a field into field-aligned coordinates
-  const Field3D toFieldAligned(const Field3D &f) {
-    return getParallelTransform().toFieldAligned(f);
+  const Field3D toFieldAligned(const Field3D &f, const REGION region = RGN_NOX) {
```

Comment:
Is there a reason for this choice of default region?

Reply:
I think the reason I did this is that:
- we (usually, probably) want to calculate the result in the y-guard cells, to avoid interpolation and needing to apply parallel boundary conditions before taking derivatives or interpolating
- we don't want to shift in the corner guard cells, because they contain invalid data and I think I was getting exceptions thrown when including them
- we didn't have a region that excludes just the corner guard cells, but includes x- and y-guard cells
- since radial derivatives should probably be done in the orthogonal coordinates rather than the field-aligned ones, the x-guard cells shouldn't normally be needed.